server/global.d.ts

```diff
@@ -12,6 +12,8 @@ import * as StreamsType from './../lib/streams';
 import * as UsersType from './users';
 import * as ChatType from './chat'
 
+import * as LotteryType from './chat-plugins/lottery';
```

Comment:
This seems unnecessary?

Reply:
As far as I can tell it's necessary for typescript to recognise it, since the file's never directly included elsewhere.

Reply:
I'm pretty sure that's because you did not include it in `tsconfig.json`.

Reply:
Yeah, I would manually include it in `tsconfig` like JetOU is suggesting.

Reply:
...right, that works